build.gradle.kts

```diff
@@ -34,14 +34,11 @@ allprojects {
     // configure which version of the annotation processor to use. defaults to the same version as the plugin
     configure<org.eclipse.edc.plugins.autodoc.AutodocExtension> {
         processorVersion.set(annotationProcessorVersion)
-        outputDirectory.set(project.buildDir)
+        outputDirectory.set(project.layout.buildDirectory.asFile.get())
     }
```

Comment:
this autodoc block is not needed as by default it is configured this way.
the `annotationProcessorVersion` could then be removed by both this file and `gradle.properties`

another thing to do is to use project `version` as `edc-build` plugin version, then `edcGradlePluginsVersion` can be removed both from this file and from `gradle.properties`

you can get some inspiration from another technology repo: https://github.com/eclipse-edc/Technology-Aws/blob/main/build.gradle.kts